docs/source/nlp/megatron_finetuning.rst

```diff
@@ -0,0 +1,57 @@
+.. _megatron_finetuning:
+
+Megatron-LM for Downstream Tasks
+================================
+
+Megatron-LM :cite:`nlp-megatron-shoeybi2019megatron` is a large, powerful transformer developed by the Applied Deep Learning Research team at NVIDIA.
+More details could be found in `Megatron-LM github repo <https://github.com/NVIDIA/Megatron-LM>`_.
+
+
+Fine-tuning
+-----------
+
+In order to fine-tune a pretrained Megatron-BERT language model on NLP downstream tasks from `NeMo/examples/nlp <https://github.com/NVIDIA/NeMo/tree/master/examples/nlp>`_, specify the ``model.language_model.pretrained_model_name`` parameter as follows:
+
+.. code-block:: bash
+
+    model.language_model.pretrained_model_name=megatron-bert-345m-uncased
+
+Available pre-trained Megatron-LM models:
+
```

Comment:
A short explanation on the difference between the megatron vs biomegataron?

Reply:
done